src/clj/puppetlabs/services/master/master_core.clj

```diff
@@ -729,7 +729,8 @@
       (schema/optional-key "job_id") schema/Int
       (schema/optional-key "transaction_uuid") schema/Str
       (schema/optional-key "environment") schema/Str
-      (schema/optional-key "options") {(schema/optional-key "capture_logs") schema/Bool}
+      (schema/optional-key "options") {(schema/optional-key "capture_logs") schema/Bool
+                                       (schema/optional-key "prefer_requested_environment") schema/Bool}
       (schema/optional-key "classes") [schema/Any]
       (schema/optional-key "parameters") {schema/Any schema/Any}}
```

Comment:
I think we wanted to remove these last two.

Reply:
I'm fine leaving that for a fixup PR.